flow/Trace.cpp

```diff
@@ -41,6 +43,19 @@
 #undef min
 #endif
 
+Reference<ITraceLogFormatter> createLogFormatter() {
+	auto f = FLOW_KNOBS->TRACE_FORMAT;
```

Comment:
I think I'd prefer that we specify the trace log format as a command line parameter. Our general philosophy around making the choice between knobs and other mechanisms (such as command line parameters or configuration through fdbcli, etc.) is that parameters we expect people to want to change shouldn't be knobs.

Reply:
I don't feel strong about this. The reason I used a knob was because adding a command line parameter is too much work because it needs to be done for every executable (I think we should eventually change this mechanism, but that's beyond the point).

Reply:
I think the last thing to be done for this is adding support for this on clients via network option. Is that something you'd be interested in adding? We could also raise it as a separate issue.

Reply:
I've opened https://github.com/apple/foundationdb/issues/997 for adding a network option for trace format. I don't mind adding it so feel free to assign it to me (it seems I can't assign it to myself currently). It's not something that's high priority for us but I'm happy to do it.